### DISCUSSION TITLE
Page loading issue

### DISCUSSION THREAD

**Discussion by naam-bhi-nhi-mila** [hay-kot/homebox#839] [Q&A] [answered]
i am trying to implement homebox on my local network of 10 pc's that don't have internet access and so after all the process of installing docker and loading the image the page is not rendering on the browser but as soon as i am providing an internet connection the page is loaded within seconds, so can anyone help me with some workaround solution for this???

**Comment by hay-kot** [comment, marked as answer]
Could you try that with the nightly tag? I believe I removed the resources that required external internet access.

**Reply by naam-bhi-nhi-mila** [reply]
I tried it and it is working now.....thank you very much
